ci/cscs-ci.yml

```
@@ -123,17 +130,45 @@ test_cscs_gh200:
   extends:
     - .container-runner-santis-gh200
     - .test_common
+  needs:
+    - build_cscs_gh200
   variables:
+    TEST_VARIANTS: 'cpu cuda12'
     GT4PY_BUILD_JOBS: 8
     # Limit test parallelism to avoid "OSError: too many open files" in the gt4py build stage.
     PYTEST_XDIST_AUTO_NUM_WORKERS: 32
+  rules:
+    - *exclude_variants_rules
+    - if: $SUBPACKAGE == 'next' && $VARIANT == 'dace' && $DETAIL == 'nomesh'
+      variables:
+        # TODO: investigate why the dace tests seem to hang with multiple jobs
+        GT4PY_BUILD_JOBS: 1
+        SLURM_TIMELIMIT: "00:15:00"
+    - when: on_success
+
+test_cscs_amd_rocm:
+  extends:
+    - .tds-container-runner-beverin-mi200
+    - .test_common
+  needs:
+    - build_cscs_amd_rocm
+  variables:
+    TEST_VARIANTS: 'cpu rocm6_0'
+    GT4PY_BUILD_JOBS: 8
+    # Limit test parallelism to avoid "OSError: too many open files" in the gt4py build stage.
+    PYTEST_XDIST_AUTO_NUM_WORKERS: 32
+    SLURM_PARTITION: mi300
+    CMAKE_PREFIX_PATH: /opt/rocm # for next
+    CUDA_HOME: /opt/rocm # for cartesian
+    SLURM_TIMELIMIT: 20 # relaxed relative to santis as there is no pressure on the queue
+  rules:
+    - *exclude_variants_rules
+    - if: $VARIANT == 'dace' && $SUBVARIANT == 'rocm6_0'
+      when: never
```

```suggestion
    - if: $VARIANT == 'dace' && $SUBVARIANT == 'rocm6_0'
      when: never
    - *exclude_variants_rules
```
skip should be applied before